### PR TITLE
Rename service to resource

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
[Deprecate `convox services`, replace with `convox resources`](https://trello.com/c/Kg8TJiCY/1102-deprecate-convox-services-replace-with-convox-resources)

File: _reference/definitions.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Definitions
+permalink: /docs/definitions/
+---
+
+# Glossary
+
+## Service
```
Comment (mattmanning) [inline _reference/definitions.md]: Let's drop the service definition from this PR and re-include it when we do the processes -> services change.

File: _reference/definitions.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+-->
+
+## Resource
+
+<div class="alert alert-warning">
+Previously called a service.
+</div>
+
+Resources are like services, but are external to your application and which your application communicates with over a network.
+Examples of typical resources used with Convox are data stores like RDS or redis, mailservers, and so on.
```
Comment (mattmanning) [inline _reference/definitions.md]: Should there be a newline here?

File: _reference/definitions.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Resources are typically stateful services.
+
+Resources correspond closely to Heroku's notion of [_backing services_](https://devcenter.heroku.com/articles/development-configuration#backing-services).
```
Comment (mattmanning) [inline _reference/definitions.md]: Can we link to https://12factor.net/backing-services here instead and mention 12factor rather than Heroku?

**Pull request merged**